components/status-bar.tsx

```diff
@@ -7,8 +7,16 @@ export default function StatusBar() {
   const {dispatch, state: {microServiceReady}} = useContext(ApplicationContext);
 
   function initialize() {
-    GithubMicroService.getHealth()
-                      .then(state => dispatch(changeMicroServiceReady(state)));
+    let timer;
```

Comment:
something tells me you're missing a rebase from master as this feature already exists